lighthouse/classes/mixins/labwhere.py

```diff
@@ -0,0 +1,40 @@
+from flask import current_app as app
+
+from lighthouse.helpers.labwhere import set_locations_in_labwhere
+from lighthouse.types import PlateEvent
+
+
+class LabWhereMixin:
+    def transfer_to_bin(self: PlateEvent) -> None:
+        """Record a transfer of the cherrypicking_source_labware to the bin
+
+        Args:
+            event (Message): The event for which to fire a callback
+
+        Returns:
+            Tuple[bool, List[str]]: True if the operation completed successfully; any errors attempting to construct the
+            message, otherwise an empty array.
+        """
+        # currently assuming only one event so only one plate_barcode
+        labware_barcodes = [self.plate_barcode]
+        location_barcode = LabWhereMixin._destroyed_barcode()
+        robot_barcode = self.robot_serial_number
+
+        set_locations_in_labwhere(
+            labware_barcodes=labware_barcodes,
+            location_barcode=location_barcode,
+            user_barcode=robot_barcode,
+        )
+
+    @staticmethod
+    def _destroyed_barcode() -> str:
```

Comment:
If we are not calling this outside this class it may not be needed to be static.